examples/MomentoRedisLambdaExample/src/MomentoRedisLambdaExample/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Amazon.CDK;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MomentoRedisLambdaExample
+{
+    sealed class Program
+    {
+        public static void Main(string[] args)
+        {
+            var app = new App();
+            new MomentoRedisLambdaExampleStack(app, "MomentoRedisLambdaExampleStack", new StackProps
+            {
+                Env = new Amazon.CDK.Environment 
+                {
+                    Account = "287427698164",
+                    Region = "us-west-2"
```

Comment:
Should extract these to parameters

Reply:
yes, we can get these values from "Aws.ACCOUNT_ID" and "Aws.REGION".